saskatoon/sitebase/templates/app/base/header.html

```diff
@@ -31,7 +32,7 @@ <h2 style="color:white">Les Fruits Défendus</h2>
                                     <li><a href={% url "logout" %}>{% translate "Logout" %}</a></li>
                                 </ul>
                             </li>
-                            {% if user.is_staff %}
+                            {% if request.user|has_group:'core' or request.user.is_staff %}
```

Comment:
as far as I understand we are able to access `user` in the templates (which should be the same thing as `request.user`) because we have enabled `'django.contrib.auth.context_processors.auth'` in `settings.py`
https://docs.djangoproject.com/en/3.0/ref/templates/api/#django-contrib-auth-context-processors-auth

I'd suggest we try to be consistent,  at least on a per template basis (`user` is used several times in header.html)

Personally I'm used to `request.user` but I'd be open to switching to `user` since we already use `perms` that relies on the same context processor. Doesn't look like there are too many:
```
./saskatoon/sitebase/templates/app/participation_create.html:11:{% if request.user.is_authenticated or harvest.is_open_to_requests == True %}
./saskatoon/sitebase/templates/app/detail_views/harvest/status.html:13:                        {% if harvest.pick_leader is None or harvest.pick_leader.id != request.user.id %}
./saskatoon/sitebase/templates/app/detail_views/harvest/status.html:102:                        {% elif harvest.pick_leader.id == request.user.id %}
```

Reply:
thanks for the explanation qynn. will edit it.